gateway/1.10/modules/ROOT/pages/policies-included-xml-threat-protection.adoc

```diff
@@ -0,0 +1,102 @@
+= XML Threat Protection Policy
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+:keywords: xml threat protection, api gateway, flex gateway, gateway, policy 
+
+[width="100%", cols="1,3"]
+|===
+>s|Policy Name| XML Threat Protection
```

Comment:
I've never seen this format!

Reply:
I'm not sure why, but all of our policies use it.